auction-app-backend/src/main/java/com/example/auctionapp/controller/ProductController.java

```diff
@@ -42,9 +42,11 @@ public ProductSearchResponse getProducts(
             @RequestParam(value = "page", defaultValue = "0") final int page,
```

Comment:
Here we can create an object that will collect all those params, and use it, we will reduce the amount of code we have